apis/discovery/v1alpha1/foreigncluster_types.go

```diff
@@ -88,14 +95,41 @@ type ForeignClusterStatus struct {
 	// +kubebuilder:validation:Optional
 	TenantControlNamespace TenantControlNamespace `json:"tenantControlNamespace"`
 
-	Outgoing Outgoing `json:"outgoing,omitempty"`
-	Incoming Incoming `json:"incoming,omitempty"`
-	// It stores most important network statuses.
-	Network Network `json:"network,omitempty"`
-	// Authentication status.
-	// +kubebuilder:validation:Enum="Pending";"Accepted";"Refused";"EmptyRefused"
-	// +kubebuilder:default="Pending"
-	AuthStatus discovery.AuthStatus `json:"authStatus,omitempty"`
+	// PeeringConditions contains the conditions about the peering related to this
+	// ForeignCluster.
+	PeeringConditions []PeeringCondition `json:"peeringConditions,omitempty"`
+}
+
+// PeeringConditionType represents different conditions that a peering could assume.
+type PeeringConditionType string
+
+// These are valid conditions of a peering.
+const (
+	// OutgoingPeeringCondition informs users about the outgoing peering status.
+	OutgoingPeeringCondition PeeringConditionType = "OutgoingPeeringCondition"
```

Comment:
Would drop the term `Condition` from the user-facing strings, as already implied by the conditions array.
```suggestion
	OutgoingPeeringCondition PeeringConditionType = "OutgoingPeering"
```